packages/smooth_app/lib/pages/onboarding/onboarding_constants.dart

```diff
@@ -0,0 +1,2 @@
+// Side padding is 8% of total width.
+double sidePadding(double screenWidth) => screenWidth * 0.08;
```

Comment:
Please don't create a 2-line file. By the way it's not a constant either.

Reply:
haha I knew this file won't fly, but thought I'll still give it a shot,